.github/workflows/documentation.yaml

```diff
@@ -28,7 +20,10 @@ jobs:
 
       - name: 👷 Generate zsd
         run: |
-          make doc
+          make doc-container
+          rc="$?"
```

Comment:
Won't GH actions exit on their own if the `1` exit code is returned? This seems redundant

Reply:
Right, it does. Will remove this.

Reply:
Done.

[78c898e](https://github.com/zdharma-continuum/zinit/pull/200/commits/78c898ea775191714c22155522da6054aa034a3e)